### ISSUE TITLE
MavenCommand.installedVersion fails with a shim

### ISSUE THREAD

**Issue by romain-grecourt** [oracle/helidon-build-tools#364]
If the `mvn` executable that is found in the PATH is a shim, `MavenCommand.installVersion` fails.

```bash
./helidon dev --debug
error: /Users/remmelt/.jenv/lib does not exist
java.lang.IllegalArgumentException: /Users/remmelt/.jenv/lib does not exist
	at io.helidon.build.util.FileUtils.assertExists(FileUtils.java:279)
	at io.helidon.build.util.FileUtils.assertDir(FileUtils.java:244)
	at io.helidon.build.util.MavenCommand.installedVersion(MavenCommand.java:157)
	at io.helidon.build.util.MavenCommand.assertRequiredMavenVersion(MavenCommand.java:196)
	at io.helidon.build.cli.impl.CommandRequirements.requireMinimumMavenVersion(CommandRequirements.java:89)
	at io.helidon.build.cli.impl.DevCommand.assertPreconditions(DevCommand.java:146)
	at io.helidon.build.cli.impl.BaseCommand.execute(BaseCommand.java:59)
	at io.helidon.build.cli.harness.CommandRunner.doExecuteCommand(CommandRunner.java:86)
	at java.util.Optional.ifPresentOrElse(Optional.java:201)
	at io.helidon.build.cli.harness.CommandRunner.doExecuteCommandName(CommandRunner.java:60)
	at java.util.Optional.ifPresentOrElse(Optional.java:201)
	at io.helidon.build.cli.harness.CommandRunner.doExecute(CommandRunner.java:50)
	at java.util.Optional.ifPresentOrElse(Optional.java:203)
	at io.helidon.build.cli.harness.CommandRunner.execute(CommandRunner.java:43)
	at io.helidon.build.cli.harness.CommandRunner.execute(CommandRunner.java:118)
	at io.helidon.build.cli.harness.CommandRunner.execute(CommandRunner.java:132)
	at io.helidon.build.cli.impl.Main.main(Main.java:36)
```

This might be a candidate for a 2.x patch release.

**Comment by romain-grecourt** [comment]
Fixed by #390

**Comment by romain-grecourt** [comment]
Needs backport to 2.x branch
